### ISSUE TITLE
[Bug] InfluxDB: Query builder detects regex to easily for measurement

### ISSUE THREAD

**Issue by candysmurf** [grafana/grafana#7276]
When data stored in influxdb as field sets, no data points shown on grafana dashboard.

Sample queries returned data in Influxdb.

`````
> select "free", "used" from "/intel/psutil/vm" where time > now() - 30m limit 5;
name: /intel/psutil/vm
----------------------
time			free		used
1484613771290085332	63647744	11691483136
1484613772301217884	58601472	11696726016
1484613773290461558	43663360	11722657792
1484613774294873045	39829504	11726950400
1484613775295781029	49172480	11718778880
`````

Please prefix your title with [Bug] or [Feature request]
For question please check [Support Options](http://grafana.org/support/). **Do not** open a github issue

Please include this information:
- What Grafana version are you using? 4.1.1
- What datasource are you using? influxdb
- What OS are you running grafana on? Mac docker
- What did you do?   view data points in Grafana
- What was the expected result?  Graph
- What happened instead? no data points

**IMPORTANT** 
If it relates to *metric data viz*:
- An image or text representation of your metric query
    
![screen shot 2017-01-16 at 5 13 14 pm](https://cloud.githubusercontent.com/assets/13841563/22004292/43ed0794-dc0f-11e6-9c5a-a10e5cb1757f.png)

- The raw query and response for the network request (check this in chrome dev tools network tab, here you can see metric requests and other request, please include the request body and request response)

select "free", "used" from "/intel/psutil/vm" where time > now() - 30m

**Comment by torkelo** [comment]
What is the response from influxdb? Check chrome dev tools network tab. 

Please use the query editor to create proper queries. You are missing a group by time which means InfluxDB can return millions of data points depending on the time range and hang your browser. A group by time with the grafana managed $interval variable is important, it makes sure there is not too many data points returned from the query (no more than one data point per pixel in the graph)

**Comment by candysmurf** [comment]
@torkelo, when I used group by which applied to aggregation, such as mean, it didn't show any data points.

**Comment by candysmurf** [comment]
@torkelo , please see the request and response:

![screen shot 2017-01-16 at 11 35 21 pm](https://cloud.githubusercontent.com/assets/13841563/22011590/c4276cee-dc44-11e6-9066-44b36373bc2f.png)

![screen shot 2017-01-16 at 11 35 32 pm](https://cloud.githubusercontent.com/assets/13841563/22011595/c7a619e2-dc44-11e6-9413-ba1886b965b6.png)

**Comment by candysmurf** [comment]
@torkelo, it didn't work with time. Please see the request and response:

![screen shot 2017-01-16 at 11 35 21 pm](https://cloud.githubusercontent.com/assets/13841563/22011650/0b923596-dc45-11e6-8dcb-0f0934d5d524.png)

![screen shot 2017-01-16 at 11 35 32 pm](https://cloud.githubusercontent.com/assets/13841563/22011657/0eded20e-dc45-11e6-91bb-147303ea4d85.png)

**Comment by torkelo** [comment]
Can you show metrics tab? Your query looks invalid, do not think your using the query builder UI because your measurements it not enclosed in quotes

**Comment by candysmurf** [comment]
@torkelo, thanks for your quick response. Please see metric tab:

![screen shot 2017-01-17 at 8 31 33 am](https://cloud.githubusercontent.com/assets/13841563/22029457/812467f8-dc8f-11e6-8a1d-6083241cbfe7.png)

**Comment by torkelo** [comment]
the `/` start symbol makes Grafana think your using a regex selector for measurement name. 

Try specifying this `/^intel\/psutil\/vm$`  (escaping the forward slashes to make it a valid regex.

**Comment by candysmurf** [comment]
@torkelo, at the mean time, if I remove the beginning of the forward slash, it would be a workaround I guess.

**Comment by torkelo** [comment]
yes

**Comment by candysmurf** [comment]
@torkelo, by removing the start forward slash, it worked very well. It has no impact on our side currently. Thanks for your help. We can just close this issue if you like.

**Comment by daniellee** [comment]
Closed by #7734
